Classes/Command/NodeIndexCommandController.php

```diff
@@ -234,131 +196,377 @@ public function indexNodeCommand(string $identifier, string $workspace = null):
      *
      * This command (re-)indexes all nodes contained in the content repository and sets the schema beforehand.
      *
-     * @param int $limit Amount of nodes to index at maximum
-     * @param bool $update if TRUE, do not throw away the index at the start. Should *only be used for development*.
+     * @param integer $limit Amount of nodes to index at maximum
```

Comment:
int / bool are the PHP type as used in method signatures, I would use them also in the comments, why do you prefer integer / boolean here?